### PR TITLE
Google Translate Feature

### PR DESCRIPTION
**Pull request by TaylorAkin**
- adds gtranslate as module
 - created custom patch that adds an overlay view style
 - adds custom scss for specific styling to our site

### CODE REVIEW

**Comment by chris-allen** [comment]
[Visit Site](https://pr-201-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-201](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-201) for lexky-d8.

File: config/sync/captcha.settings.yml [hunk 1]
```diff
@@ -1,12 +1,10 @@
 _core:
   default_config_hash: pfrv6lPS72f5WG1UB4wHkgQV2invlHwVpaCy_mY-ClQ
-enabled_default: 0
 default_challenge: captcha/Math
 description: 'This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.'
 administration_mode: false
 allow_on_admin_pages: false
 add_captcha_description: true
-wrong_captcha_response_message: 'The answer you entered for the CAPTCHA was not correct.'
```
Comment (chris-allen) [inline config/sync/captcha.settings.yml:9]: Not sure why this file was touched. Any ideas?

File: config/sync/honeypot.settings.yml [hunk 2]
```diff
@@ -1,25 +1,12 @@
...
-time_limit: 5
-expire: 300
+time_limit: 0
 form_settings:
   user_register_form: false
   user_pass: false
   contact_message_feedback_form: true
   contact_message_page_feedback_form: true
   contact_message_personal_form: false
-  node_article_form: false
```
Comment (chris-allen) [inline config/sync/honeypot.settings.yml:12]: Again. Curious why this was touched

File: web/themes/custom/lex/patterns/components/header.twig [hunk 3]
```diff
@@ -17,7 +17,6 @@
             <span class="hamburger-line"></span>
             <span class="hamburger-line"></span>
           </div>
-          {# <span class="menu-label">Search</span> #}
```
Comment (chris-allen) [inline web/themes/custom/lex/patterns/components/header.twig:20]: Was this a requirement of google translate. Or is this just tweaks during iterating on the multidev environment?

**Review by chris-allen — CHANGES_REQUESTED** [review]

File: config/sync/captcha.settings.yml (hunk 1 above)
Comment (TaylorAkin) [reply config/sync/captcha.settings.yml:9]: It was added when bouncing between branches. Needs to be tossed out.

File: config/sync/honeypot.settings.yml (hunk 2 above)
Comment (TaylorAkin) [reply config/sync/honeypot.settings.yml:12]: It was added when bouncing between branches. Needs to be tossed out.

File: web/themes/custom/lex/patterns/components/header.twig (hunk 3 above)
Comment (TaylorAkin) [reply web/themes/custom/lex/patterns/components/header.twig:20]: They are commented out lines. I just removed them to remove them.

File: web/themes/custom/lex/scss/regions/_takeover_nav.scss [hunk 4]
```diff
@@ -35,13 +35,15 @@ div.fade, div.search-fade {
 .navigation--top-level {
   color: white;
   display: inline-block;
+  // overflow: scroll !important;
```
Comment (TaylorAkin) [inline web/themes/custom/lex/scss/regions/_takeover_nav.scss:38]: This is what fixes styling on PC/Linux when opening the hamburger menu.

**Review by chris-allen — APPROVED** [review]

**Pull request merged**